experimental/LieAlgebras/src/root_systems.jl

```diff
@@ -0,0 +1,224 @@
+mutable struct RootSystem
+  roots::Vector{Vector{Int}}
+  simple_roots::Vector{Vector{Int}}
+  positive_roots::Vector{Vector{Int}}
+  root_system_type::Tuple{Symbol,Int64}
```

Comment:
```suggestion
  root_system_type::Tuple{Symbol,Int}
```
One should use `Int` and let the architecture decide on the concrete type (if the application doesn't need one particular size). Here and the next few comments.